### ISSUE TITLE
Inconsistent indentation errors, 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by gryphonmyers** [elarkham/VGMdb-BeetsPlug#1]
When I install the plugin, beets complains about inconsistent indentation on line 56. I was able to make it stop complaining about the indentation errors by reindenting various lines, but then I got this other error when running an import:

Traceback (most recent call last):
  File "/usr/local/bin/beet", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/site-packages/beets/ui/__init__.py", line 1209, in main
    _raw_main(args)
  File "/usr/local/lib/python3.5/site-packages/beets/ui/__init__.py", line 1196, in _raw_main
    subcommand.func(lib, suboptions, subargs)
  File "/usr/local/lib/python3.5/site-packages/beets/ui/commands.py", line 930, in import_func
    import_files(lib, paths, query)
  File "/usr/local/lib/python3.5/site-packages/beets/ui/commands.py", line 907, in import_files
    session.run()
  File "/usr/local/lib/python3.5/site-packages/beets/importer.py", line 319, in run
    pl.run_parallel(QUEUE_SIZE)
  File "/usr/local/lib/python3.5/site-packages/beets/util/pipeline.py", line 445, in run_parallel
    six.reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/usr/local/lib/python3.5/site-packages/six.py", line 686, in reraise
    raise value
  File "/usr/local/lib/python3.5/site-packages/beets/util/pipeline.py", line 312, in run
    out = self.coro.send(msg)
  File "/usr/local/lib/python3.5/site-packages/beets/util/pipeline.py", line 194, in coro
    func(*(args + (task,)))
  File "/usr/local/lib/python3.5/site-packages/beets/importer.py", line 1261, in lookup_candidates
    task.lookup_candidates()
  File "/usr/local/lib/python3.5/site-packages/beets/importer.py", line 589, in lookup_candidates
    autotag.tag_album(self.items, search_ids=self.search_ids)
  File "/usr/local/lib/python3.5/site-packages/beets/autotag/match.py", line 457, in tag_album
    va_likely):
  File "/usr/local/lib/python3.5/site-packages/beets/plugins.py", line 496, in decorated
    for v in generator(*args, **kwargs):
  File "/usr/local/lib/python3.5/site-packages/beets/autotag/hooks.py", line 596, in album_candidates
    for candidate in plugins.candidates(items, artist, album, va_likely):
  File "/usr/local/lib/python3.5/site-packages/beets/plugins.py", line 355, in candidates
    for candidate in plugin.candidates(items, artist, album, va_likely):
TypeError: 'NoneType' object is not iterable

**Comment by elarkham** [comment]
Ah the tabs thing is embarrassing, not sure how I missed those. I just pushed a fix for that. I kinda just threw this plugin together in about a day so try not to expect too much of it. I was able to use it for about 300 album imports without issue though so I don't exactly think its _that bad_. 

As for your traceback, I'm not super sure whats wrong since it's not really tracing back to my plugin (though it could be related) and I cannot reproduce this. Also are you running this with Python 3.5 or is that some kind of compatibility thing? Beets does not yet support Python 3.0 and above so I can't help but ask. 

This is my `beets --version` output if it helps at all:
`beets version 1.4.0`
`python version 2.7.12`
`plugins: discogs, duplicates, edit, embedart, fetchart, fromfilename, fuzzy, importadded, info, inline, lastgenre, mbsync, missing, replaygain, vgmdb`

**Comment by gryphonmyers** [comment]
No worries, I figured this was WIP. It's possible I messed something up
when I was fixing your indentation. Let me try again tonight. Thanks for
the quick response!

**Comment by gryphonmyers** [comment]
Hmm ok so I don't get that error anymore, but I'm not seeing any vgmdb results show up when I try to import releases that I know exist there (using the exact album name). Would the python version really be a problem? I am on python 3.5.2 but I thought that was ok since the beets install instructions say "You will need Python. Beets works on Python 2.7 and Python 3.4 or later."

**Comment by elarkham** [comment]
Huh, didn't know it worked with Python 3.4+ until now, this [blog post](http://beets.io/blog/py3k.html) made me think otherwise but I see now that it's actually 5 years old... This [wiki post](https://github.com/beetbox/beets/wiki/Python3) shows they are working on actually getting a full port done, figures they made this only 8 days after I made the dang thing. Would have written it in Python 3 had I known. 

That being said, I'm not 100% sure this is even related to Python 3, but if it is then you can fix it by running beets with Python 2 instead of Python 3. 

What OS/Distro are you using? Also could you try importing an album manually via id? I'm curious if that works.

**Comment by gryphonmyers** [comment]
Well I just went through the long, confusing and tiring process of installing and configuring pyenv, then getting beets on 2.7.13 and your plugin now works. So apparently it is incompatible with 3.5. I might take a stab at figuring out what API updates are necessary but for now it works. Closing this issue, but you might want to note the compatibility issue in the readme. Thanks for your help!
